src/ui/Avatar/Avatar.tsx

```diff
@@ -50,18 +62,29 @@ export const Avatar = ({
   path,
   name,
   onClick,
+  orgLogo,
   ...props
 }: {
   path?: string;
   /** User's name is used as a fallback in case of failing to load avatar. */
   name?: string;
   onClick?: () => void;
+  orgLogo?: boolean;
```

Comment:
I think you can do without this boolean. It would be good for this component to stay generic and just know how to render an image or a fallback, but not what object the image is for.

1. getAvatarPath will just pass through the string already if it starts with `https://`
2. `...props` already passes through `size`, so you can just set `size` as an outer prop (i.e. in OrgLogoUpload)